### PR TITLE
update host selection integration

### PR DESCRIPTION
**Pull request by rawagner**
depends on https://github.com/openshift-assisted/assisted-ui-lib/pull/776

### CODE REVIEW

File: src/components/cluster-deployments/transitionCallbacks.ts [hunk 1]
```diff
@@ -263,53 +258,42 @@ export const getOnSaveHostsSelection =
...
               },
-            ]);
-          }),
+            },
+          ]);
+        }),
       );
 
       // TODO(mlibra): check for errors in the releasedResults and reservedResults
-      console.log('--- releaseResults (TODO): ', releaseResults);
```
Comment (mareklibra) [inline src/components/cluster-deployments/transitionCallbacks.ts:298]: How is this handled?

Comment (rawagner) [reply src/components/cluster-deployments/transitionCallbacks.ts:298]: you mean errors ? Its not handled. TODO still applies, i just removed the logging

Comment (mareklibra) [reply src/components/cluster-deployments/transitionCallbacks.ts:298]: these remainers were tracked just via those `TODO`s so far, they will be forgotten when just removing

Comment (rawagner) [reply src/components/cluster-deployments/transitionCallbacks.ts:298]: but we have a TODO comment above it 

```
// TODO(mlibra): check for errors in the releasedResults and reservedResults
```

why do we need console.logs ? Its just unnecessary noise getting through the code

Comment (rawagner) [reply src/components/cluster-deployments/transitionCallbacks.ts:298]: but I can return the logging if you prefer :)

**Review by mareklibra — APPROVED** [review]

**Pull request merged**